<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDialog</class>
 <widget class="QDialog" name="SelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>239</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="firstHandButton">
      <property name="text">
       <string>先手</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="secondHandButton">
      <property name="text">
       <string>后手</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="easyButton">
      <property name="text">
       <string>简单</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="middleButton">
      <property name="text">
       <string>中等</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="hardButton">
      <property name="text">
       <string>困难</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ackButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
